src/CommonDBTM.php

```diff
@@ -500,12 +500,11 @@ public function post_getFromDB()
 
     public function getFormFields(): array
     {
-        // NOTE: Post code field named differently for Suppliers. Placed after town to maintain field order from 9.5
```

Comment:
Did not understand that one. The is no "postalcode" field in dtaabase, in main, 10.0/bf nor 9.5/bf branches.
On template side, this appears in `templates/generic_show_form.html.twig` with no reason I can see.

Reply:
I searched the GLPI code all the way back to 0.20 and the only time "postalcode" shows is in 10.0 in this template. I am clueless too.